derrida/books/templates/books/public_canvas_detail.html

```diff
@@ -353,7 +353,8 @@ <h3 class="gallery-image-view__list-title--bold">Admin Actions</h3>
 
         </div>
         {# deep zoom #}
-        <div id="deep-zoom"></div>
+        <a name="zoom"> </a>
```

Comment:
maybe a dumb question, but what does `name` on `<a>` do? what are these links for?

Reply:
It's an anchor for a link. pa11y was complaining because we have links for `#annotation` and `#zoom` but no targets. I couldn't use id because they didn't match and I didn't want to break any of the javascript.

Reply:
ah, makes sense! thank you